example/tests/tests.dart

```diff
@@ -1,100 +1,258 @@
 import 'dart:async';
-import 'dart:io';
+import 'dart:ui';
```

Comment:
Feel free to revert this. The problem in the previous was that `dart:io` is not compatible with web. Also the `std*()` didn't output correctly in the Android console.